backend/resources/teams.py

```diff
@@ -624,3 +624,26 @@ def delete(self, team_id, user_id):
         response = Response()
         response.status_code = 204
         return response
+
+
+class TeamAnswersRes(Resource):
+
+    @roles_allowed(['admin'])
+    def get(self, team_id):
+        answers = Answer.query.filter_by(team_id=team_id).all()
+        response = jsonify(list(map(lambda answer: answer.serialize(), answers)))
```

Comment:
It's more common to use list comprehension instead of list/filter. So, `[ans.serialize() for ans in answers]`

Reply:
done